man/sd_bus_slot_get_bus.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="sd_bus_slot_get_bus" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>sd_bus_slot_get_bus</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_slot_get_bus</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_slot_get_bus</refname>
+    <refname>sd_bus_slot_get_current_handler</refname>
+    <refname>sd_bus_slot_get_current_message</refname>
+    <refname>sd_bus_slot_get_current_userdata</refname>
+
+    <refpurpose>Query information of a bus slot object</refpurpose>
```

Comment:
"of" sounds wrong. Maybe this can be more precise with "Query the bus and other objects associated with a slot object"?

Reply:
I went with `Query information attached to a bus slot object`. Does that work?

Reply:
Yep, looks nice, thanks.